swan-lake/learn/writing-secure-ballerina-code.md

```diff
@@ -659,7 +659,7 @@ curl -k -v https://localhost:9091/hello
 Authentication failure
 ```
 
-Once a request is made with a valid, authentication information, but if the introspection endpoint does not respond with the "scope" attribute of the response JSON payload or respond with the "scope" attribute, which are not the expected scopes, an authorization failure will occur.
+Once a request is made with valid authentication information, but if the introspection endpoint does not respond with the `scope` attribute of the response JSON payload or respond with the `scope` attribute, which is not the expected scope, an authorization failure will occur.
```

Comment:
```suggestion
If a request is made with valid authentication information, but the introspection endpoint does not respond with the `scope` attribute of the response JSON payload or respond with the `scope` attribute, which is not the expected scope, an authorization failure will occur.
```

Reply:
Same comment previously mentioned with the repetition of `scope` attribute

Reply:
Fixed.